dspace-api/src/test/resources/org/dspace/identifier/handle-forms.properties

```diff
@@ -0,0 +1,5 @@
+# Key is the identifier to be tested; value is "true" or "false":  should this
+# identifier be supported by the provider under test?
+
+http://dx.doi.org/10.14279/depositonce-5383 = false
+123456789/1 = true
```

Comment:
@mwoodiupui : I agree. Which is why I wanted to point out *why* that test is successful.  Either we need to build this into a better test, or we should minimally add a comment to that `handle-form.properties` file to explain why that test is successful (in case someone should purposefully update the `local.cfg` for the Test Framework only to find this test failing)